lib/bench/Bench/Pos/Criterion/FollowTheSatoshiBench.hs

```diff
@@ -20,8 +20,11 @@ type UtxoSize = Int
 
 -- [CSL-192]: make a trick and use special unsafe arbitrary instances
 -- for generation of such things
-arbitraryUtxoOfSize :: UtxoSize -> Gen Utxo
-arbitraryUtxoOfSize n = fromList . take n <$> infiniteListOf arbitraryUnsafe
+arbitraryUtxoOfSize :: UtxoSize -> Gen [(StakeholderId, Coin)]
+arbitraryUtxoOfSize n = take n <$> infiniteListOf arbitraryUnsafe
+
+-- block/src/Pos/Lrc/FtsPure.hs:
+-- followTheSatoshi :: HasConfiguration => SharedSeed -> [(StakeholderId, Coin)] -> SlotLeaders
```

Comment:
Okay, I see why you made these changes, looks good. But in general there shouldn't be commented out code, so can you please remove it?